packages/p2p/src/components/verification/verification.jsx

```diff
@@ -25,9 +25,9 @@ const Verification = observer(() => {
                     <Localize i18n_default_text='Please register with us!' />
                 </div>
                 <div className='p2p-verification__text-description'>
-                    <p>
+                    <Text as='p'>
                         <Localize i18n_default_text='To use DP2P, you need to choose a display name (a nickname) and verify your identity.' />
-                    </p>
+                    </Text>
```

Comment:
Correct me if I'm wrong, but i think we need to pass css properties of `p2p-verification__text-description` to `Text`..

Reply:
Yes we can transfer the style of div to the p. Thanks for mentioning.